AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -333,6 +333,7 @@ forbes.com#@#.AD-label300x250
 forbes.com#@#.AD-Rotate
 !
 mail.yahoo.com##div[aria-describedby="adblock-whitelist-cue"]
+||prepareplanes.com^$third-party
```

Comment:
Not requested when I visit the site.
Only seeing they check these elements (but not reproduced adblock message):
 cssClass: "pub_300x250 pub_300x250m pub_728x90 text-ad textAd text_ad text_ads text-ads text-ad-links"

Reply:
Try to visit with american IP
<details>
<img src="https://user-images.githubusercontent.com/18536328/82455186-5727a800-9abb-11ea-9234-243b06a0f060.png">
</details>